packages/service-library/src/servicelib/monitoring.py

```diff
@@ -25,80 +25,93 @@ def middleware_factory(app_name):
+                request.remote,
+                request.method,
+                request.path,
                 resp_time,
-                resp.status
+                resp.status,
             )
         finally:
             # metrics on the same request
-            resp_time = time.time() - request['start_time']
-            request.app['REQUEST_LATENCY'].labels(
-                app_name, request.path).observe(resp_time)
+            resp_time = time.time() - request["start_time"]
+            request.app["REQUEST_LATENCY"].labels(app_name, request.path).observe(
+                resp_time
+            )
 
-            request.app['REQUEST_IN_PROGRESS'].labels(
-                app_name, request.path, request.method).dec()
+            request.app["REQUEST_IN_PROGRESS"].labels(
+                app_name, request.path, request.method
+            ).dec()
 
-            request.app['REQUEST_COUNT'].labels(
-                app_name, request.method, request.path, resp.status).inc()
+            request.app["REQUEST_COUNT"].labels(
+                app_name, request.method, request.path, resp.status
+            ).inc()
 
         return resp
 
     middleware_handler.__middleware_name__ = __name__
     return middleware_handler
 
+
 async def metrics(_request):
     # TODO: NOT async!
     # prometheus_client access to a singleton registry!
     resp = web.Response(body=prometheus_client.generate_latest())
     resp.content_type = CONTENT_TYPE_LATEST
     return resp
 
+
 async def check_outermost_middleware(app: web.Application):
     m = app.middlewares[0]
-    ok = m and hasattr(m, "__middleware_name__") and m.__middleware_name__==__name__
+    ok = m and hasattr(m, "__middleware_name__") and m.__middleware_name__ == __name__
     if not ok:
         # TODO: name all middleware and list middleware in log
-        log.critical("Monitoring middleware expected in the outermost layer."
-            "TIP: Check setup order")
+        log.critical(
+            "Monitoring middleware expected in the outermost layer."
+            "TIP: Check setup order"
+        )
+
 
 def setup_monitoring(app: web.Application, app_name: str):
     # NOTE: prometheus_client registers metrics in **globals**. Therefore
     # tests might fail when fixtures get re-created
 
     # Total number of requests processed
-    app['REQUEST_COUNT'] = Counter(
-        'http_requests_total', 'Total Request Count',
-        ['app_name', 'method', 'endpoint', 'http_status']
+    app["REQUEST_COUNT"] = Counter(
+        "http_requests_total",
+        "Total Request Count",
+        ["app_name", "method", "endpoint", "http_status"],
     )
 
     # Latency of a request in seconds
-    app['REQUEST_LATENCY'] = Histogram(
-        'http_request_latency_seconds', 'Request latency',
-        ['app_name', 'endpoint']
+    app["REQUEST_LATENCY"] = Histogram(
+        "http_request_latency_seconds", "Request latency", ["app_name", "endpoint"]
```

Comment:
Whats the rule here? Above and below you have new lines after the first two parameters

Reply:
It is optimized to outline differences in diffs. The width and height can be slightly changed.
The important thing here is that with one key-stroke all the code is formatted uniformly